mars/dataframe/expressions/arithmetic/core.py

```diff
@@ -585,3 +587,226 @@ def __call__(self, x1, x2):
 
     def rcall(self, x1, x2):
         return self._call(x1, x2)
+
+class DataFrameUnaryOpMixin(DataFrameOperandMixin):
+    __slots__ = ()
+
+    @classmethod
+    def _tile_dataframe(cls, op):
+        df = op.outputs[0]
+        operand = op.inputs[0]
```

Comment:
Assume we have `lhs` and `rhs` attributes, we can get input df by judging if it's a `Base`.

```
in_df = op.lhs if isinstance(op.lhs, Base) else op.rhs
out_df = op.outputs[0]
```